### DISCUSSION TITLE
希望侧加载提交一下winget的包

### DISCUSSION THREAD

**Discussion by pheotry** [Richasy/Bili.Uwp#910] [想法]
win11已经内置了winget，如果可以提交的话，以后安装更新在终端里一条命令就行了，很方便，也不需要每次打开浏览器下载了

既然有商店版本了，那winget应该可以直接装商店版，不过侧加载版本还是希望可以再次提交一份到winget

**Comment by Richasy** [comment]
已经可以在商店下载的情况下，不会就侧加载单独提供一份包。
